src/commands/sites/sites-create-template.js

```diff
@@ -0,0 +1,251 @@
+// @ts-check
+
+const slugify = require('@sindresorhus/slugify')
+const inquirer = require('inquirer')
+const pick = require('lodash/pick')
+const sample = require('lodash/sample')
+const fetch = require('node-fetch')
+const prettyjson = require('prettyjson')
+const { v4: uuidv4 } = require('uuid')
+
+const { chalk, error, log, logJson, track, warn } = require('../../utils')
+const { getGitHubToken } = require('../../utils/gh-auth')
+
+const SITE_NAME_SUGGESTION_SUFFIX_LENGTH = 5
+
+let ghToken
+
+// Templates are hardcoded for now
+const templates = [
+  {
+    name: 'Next.js Blog theme',
+    sourceCodeUrl: 'https://github.com/netlify-templates/nextjs-blog-theme',
+    slug: 'netlify-templates/nextjs-blog-theme',
+  },
+  {
+    name: 'Next.js Starter',
+    sourceCodeUrl: 'https://github.com/netlify-templates/next-netlify-starter',
+    slug: 'netlify-templates/next-netlify-starter',
+  },
+  {
+    name: 'Nuxt.js Starter',
+    sourceCodeUrl: 'https://github.com/Gomah/bluise',
+    slug: 'Gomah/bluise',
+  },
+  {
+    name: 'Hugo Blog',
+    sourceCodeUrl: 'https://github.com/netlify-templates/one-click-hugo-cms',
+    slug: 'netlify-templates/one-click-hugo-cms',
+  },
+]
+
+/**
+ * The sites:create-template command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const sitesCreate = async (options, command) => {
+  const { api } = command.netlify
+
+  await command.authenticate()
+
+  console.log(
```

Comment:
please use our `log` function instead of `console.log` you've already imported it from utils otherwise it will collide with the `--json` option

```suggestion
 log(
```